src/Utopia/Messaging/Adapters/SMS/GEOSMS.php

```diff
@@ -0,0 +1,289 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Adapters\SMS as SMSAdapter;
+use Utopia\Messaging\Messages\SMS;
+
+class GEOSMS extends SMSAdapter
+{
+    protected const COUNTRY_CODES = [
+        '44' => 'UK (+44)',
+        '1' => 'USA (+1)',
+        '213' => 'Algeria (+213)',
+        '376' => 'Andorra (+376)',
+        '244' => 'Angola (+244)',
+        '1264' => 'Anguilla (+1264)',
```

Comment:
Can we use constants here like we do for appwrite errors ? https://github.com/appwrite/appwrite/blob/main/app/config/errors.php#L11-L16

Then we can use constants to setup the different local's ( or locales ) like so for better dev ex and maintainability 
```php
$sms
    ->setLocal(Local::India, new MSG91())
    ->setLocal('+44', new Vodafone())
```